tests/trading/test_makeOrder.py

```diff
@@ -155,7 +155,7 @@ def test_makeOrder_failure(contractsFixture):
 
     # makeOrder exceptions (pre-placeBid/placeAsk)
     with raises(TransactionFailed):
-        makeOrder.publicMakeOrder(ASK, fix('1'), fix('0.6'), market.address - 1, YES, 0, 0, 42, sender=tester.k1)
+        makeOrder.publicMakeOrder(ASK, fix('1'), fix('0.6'), market.address, YES, 0, 0, 42, sender=tester.k1)
```

Comment:
I believe this should fail in this case, since `market.address` is a valid market, `publicMakeOrder` should not throw.  I'm surprised this change didn't break the test (suggesting the test has another problem).